inbm/dockerfiles/Dockerfile-Windows.m4

```diff
@@ -1,7 +1,7 @@
 include(`image.main.m4')
 
 # base windows/wine build image
-FROM registry.hub.docker.com/batonogov/pyinstaller-windows:python-3.10 as base-windows
+FROM registry.hub.docker.com/batonogov/pyinstaller-windows:3.1.0 as base-windows
```

Comment:
(Python 3.11)

Reply:
Should it be 3.11 instead of 3.1?  Or do they do something weird here

Reply:
@nmgaston this is 3.11